packages/renderer/src/lib/models/SemanticRouterCreate.svelte

```
@@ -0,0 +1,186 @@
+  } catch (err: unknown) {
+    error = err instanceof Error ? err.message : String(err);
+  } finally {
+    saving = false;
+  }
+}
+
+function parseTimeout(value: string): number | undefined {
+  const trimmed = value.trim();
+  if (!trimmed) return undefined;
+  const match = /^(\d+)\s*s?$/.exec(trimmed);
+  if (match) return parseInt(match[1], 10);
```

Biome (2.4.16)

[error] Use Number.parseInt instead of the equivalent global.

(lint/style/useNumberNamespace)